spec/calendar.html

```diff
@@ -186,6 +186,25 @@ <h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-temporal-maybecreatecalendarmethodsrecordfromrelativeto" type="abstract operation">
+      <h1>
+        MaybeCreateCalendarMethodsRecordFromRelativeTo (
+          _plainRelativeTo_: a `Temporal.PlainDate` or *undefined*,
+          _zonedRelativeTo_: a `Temporal.ZonedDateTime` or *undefined*,
+          _methods_: a List of one of ~dateAdd~, ~dateFromFields~, ~dateUntil~, ~day~, ~fields~, ~mergeFields~, ~monthDayFromFields~, or ~yearMonthFromFields~,
+        ): either a normal completion containing either a Calendar Methods Record or *undefined*, or a throw completion
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>It creates a Calendar Methods Record from the calendar of the given `relativeTo` object, looking up the methods given in _methods_. If no `relativeTo` object is given, it returns *undefined* instead of a Calendar Methods Record.</dd>
+      </dl>
+      <emu-alg>
+        1. If _zonedRelativeTo_ is *undefined* and _plainRelativeTo_ is *undefined*, return *undefined*.
+        1. If _zonedRelativeTo_ is not *undefined*, let _calendar_ be _zonedRelativeTo_.[[Calendar]]; else let _calendar_ be _plainRelativeTo_.[[Calendar]].
```

Comment:
Nit: I think having a dedicated operation (and an alias for the list of field names) allows this logic to better highlight the similar treatment of plain vs. zoned input.
```
        1. If _zonedRelativeTo_ is not *undefined*, return ? CreateCalendarMethodsRecord(_zonedRelativeTo_.[[Calendar]], _methods_).
        1. If _plainRelativeTo_ is not *undefined*, return ? CreateCalendarMethodsRecord(_plainRelativeTo_.[[Calendar]], _methods_).
        1. Return *undefined*.
```

Reply:
Agreed, I like this better.